app/javascript/packs/application.js

```diff
@@ -1,7 +1,27 @@
+import 'babel-polyfill'
+
 import 'jquery' // jQuery has been configured to be globally available
 
 // Non-React code
 import '../modules/navigation'
 
 // React
-import '../App'
+import '../App/startup'
+
+import scrollHelper from '../App/scrollHelper'
+import getScreenSize from '../App/lib/screenSize'
+import debounce from 'debounce'
+
+const setScreenSize = () => {
+  window.screenSize = getScreenSize()
+  window.isMobile = !['m', 'l'].includes(window.screenSize)
+  window.isTablet = window.screenSize === 'm'
+  window.isDesktop = window.screenSize === 'l'
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  setScreenSize()
+})
+
+window.addEventListener('resize', debounce(setScreenSize, 400))
+window.addEventListener('resize', debounce(scrollHelper, 400))
```

Comment:
I was thinking about to add it into some react component. But it's not related to react app. So, I just moved it here from old `App/index.js`.